src/components/CookieRejectModal.js

```diff
@@ -0,0 +1,18 @@
+import _ from 'lodash/fp'
+import { h } from 'react-hyperscript-helpers'
+import Modal from 'src/components/Modal'
+import { useStore } from 'src/libs/react-utils'
+import { authStore } from 'src/libs/state'
+
+
+const CookieRejectModal = () => {
+  const { cookiesAccepted } = useStore(authStore)
+  return cookiesAccepted === false && h(Modal, {
+    title: 'Cookies are required for Terra',
+    showCancel: false,
```

Comment:
Note that you can still dismiss the dialog by pressing Escape (it seems like you are trying to prevent the dialog being dismissed, but it's not clear to me what you intend the experience to be for someone who does not want to accept the cookies).

Reply:
So, the short version is that we have said that you can't use the site without accepting cookies. Really the intended experience for someone who does not want to accept the cookies is, sadly, to just close the site atm. We, as a Terra engineering team, probably need to have a discussion about what parts of the site you _can_ use without cookies, but either way, this is better than we have now.

Reply:
It is better now that Escape and clicking outside do not dismiss the dialog.